python/tvm/relay/op/strategy/x86.py

```diff
@@ -52,7 +52,7 @@ def schedule_reduce_cpu(attrs, outs, target):
 def schedule_concatenate_cpu(attrs, outs, target):
     """schedule concatenate op for x86"""
     with target:
-        return topi.x86.schedule_concatenate(outs)
+        return topi.transform.schedule_concatenate(outs)
```

Comment:
why need this change?

Actually, since we define `concatenate_strategy_cpu` below, do we need `schedule_concatenate_cpu` here?